src/test/common/config.js

```diff
@@ -7,11 +7,13 @@ const SESSION_DETAILS_BASE_URL = process.env.SESSION_DETAILS_BASE_URL || `http:/
 const SESSION_DETAILS_PATH = '/session-details/confirmation-code'
 
 module.exports.PORT = process.env.PORT
-module.exports.TEST_BROWSER = process.env.TEST_BROWSER || 'chrome'
+module.exports.TEST_BROWSER = 'chrome'
```

Comment:
I'll revert this once all branches have inherited this and we can remove the env setting in travis